notebooks/01-lightcurves/01-Lightcurves.ipynb

```diff
@@ -56,9 +56,11 @@
    "source": [
```

Comment:
<p>thoughts on removing this homework exercise, given the difficulty of background subtraction? I think it's ok for a "simpler" lesson like this to omit hw</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>commented it out</p>